aemedge/blocks/author-profile/author-profile.js

```diff
@@ -37,23 +38,35 @@ async function getAuthorEntry(entryFilter) {
   return (!result || result.length < 1) ? null : completeEntry(result[0]);
 }
 
-function renderProfile(entry) {
+function asEntry(authorName) {
```

Comment:
This can go as reusable functions in utils.js

Reply:
It is now in /libs/profile/profile.js, as it is related to authors and profiles.